app/views/pages/home.json.jbuilder

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
```

Comment:
I don't see why not.

On another note, it might be best if we split the APIs for authentication. This here is just for loading the global values. I named it `#home` just because it's originally used in the home page, but not necessarily so.

Ideally, it'd be best if (1) `courses`, (2) `isSuperAdmin` and `isInstanceAdmin`, and (3) the authentication states you mentioned are all separate APIs. Easier to name, and better separation of concern. But (3) isn't here yet, and I'm lazy to open those APIs now LOL. Maybe we can split this when we do the authentication. We'll have to figure out how to handle _first-load information_ for the new client anyway.